src/components/views/dialogs/InviteDialog.tsx

```diff
@@ -1099,6 +1413,10 @@ export default class InviteDialog extends React.PureComponent<IInviteDialogProps
                 onToggle={this._toggleMember}
                 highlightWord={this.state.filterText}
                 isSelected={this.state.targets.some(t => t.userId === r.userId)}
+                isFavorite={(this.state.favorites?.indexOf(r.user.name)  !== -1 ? true: false) || false}
```

Comment:
Please remove 
? true: false 
as it is redundant

Reply:
This one has been actioned.